### DISCUSSION TITLE
useExpanded with useGlobalFilter

### DISCUSSION THREAD

**Discussion by agneyaloya** [TanStack/table#2491] [General]
I'm trying to implement a Kitchen sink of my own with data that's completely nested under a single root node. I'm able to feed the data so that useExpanded is working but my Global Search Filter now only does searching on the root node. It doesn't identify the children.

Question: Is this a limitation in react-table out-of-the-box? Any pseudo-code/ high-level recommendations to code around this?

**Comment by ardiz90** [comment]
I'm also interested in this problem, can someone help?
